includes/Core/Storage/Post_Meta_Setting.php

```diff
@@ -115,12 +128,11 @@ protected function get_sanitize_callback() {
 	 * @return mixed Value set for the setting, or default if not set.
 	 */
 	public function get( $post_id ) {
-		$value = $this->post_meta->get( $post_id, static::META_KEY, true );
-		if ( false === $value ) {
-			$value = $this->get_default();
+		if ( $this->has( $post_id, static::META_KEY ) ) {
```

Comment:
This should be `! has` for returning the default 😄 

Also, this should only need to pass through the `$post_id`.

Reply:
:facepalm: yes, updated.